pyrightconfig.stricter.json

```diff
@@ -88,6 +88,7 @@
         "stubs/reportlab",
         "stubs/requests",
         "stubs/requests-oauthlib",
+        "stubs/rfc3339-validator",
```

Comment:
This entry shouldn't be necessary, as the library is fully annotated.

Reply:
got it, thank you